app/models/game.rb

```diff
@@ -1,10 +1,35 @@
 class Game < ActiveRecord::Base
 
-  has_many :memberships
-  has_many :users, through: :memberships
+  validates :lobby, presence: true, uniqueness: true
 
-  def add_user(user)
-    memberships.create user: user
+  belongs_to :lobby
+
+  before_create :setup_game
+
+  CARD_ATTRIBUTES = {
+    symbol:  [ :diamond, :sguiggle, :oval ],
+    number:  [ :one, :two, :three ],
+    color:   [ :red, :green, :purple ],
+    shading: [ :solid, :striped, :open ]
+  }
+
+  def deck
+    dealer.get_cards(read_attribute(:deck))
+  end
+  
+  def board
+    dealer.get_cards(read_attribute(:board))
+  end
+
+  private
+
+  def dealer
+    @dealer ||= Dealer.new(CARD_ATTRIBUTES)
+  end
+
+  def setup_game
+    self.deck  = dealer.new_deck.map(&:id)
+    self.board = read_attribute(:deck).pop(12)
```

Comment:
In an integer field? I don't think I follow...

Regardless this method now looks like `12.times { game.board << game.deck.pop }` in the other feature branch since `#pop(n)` returns a new array, while `#pop` is destructive.